src/test/java/com/basho/riak/client/api/commands/itest/ITestTimeSeries.java

```diff
@@ -264,8 +322,24 @@ public void TestDeletingWithNotFoundKeyDoesNotReturnError() throws ExecutionExce
         final RiakFuture<Void, String> deleteFuture = client.executeAsync(delete);
 
         deleteFuture.await();
-        assertTrue(deleteFuture.isSuccess());
-        assertNull(deleteFuture.cause());
+        assertFutureSuccess(deleteFuture);
+    }
+
+    @Test
+    public void TestDescribeTable() throws InterruptedException, ExecutionException
+    {
+        RiakClient client = new RiakClient(cluster);
+
+        Query query = new Query.Builder("DESCRIBE " + tableName).build();
+
+        final RiakFuture<QueryResult, String> resultFuture = client.executeAsync(query);
+
+        resultFuture.await();
+        assertFutureSuccess(resultFuture);
+
+        final QueryResult tableDescription = resultFuture.get();
+        assertEquals(7, tableDescription.getRowsCount());
```

Comment:
Might be a good idea to verify it completely to fix the result format in this test